packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -265,7 +265,7 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
   }) as unknown as DefineComponent<NuxtLinkProps>
 }
 
-export default defineNuxtLink({ componentName: 'NuxtLink' })
+export default /* @__PURE__ */ defineNuxtLink({ componentName: 'NuxtLink' })
```

Comment:
Out of curiosity, is the `@` instead of the `#` intended here? ☺️

Reply:
it is not! thank you!

Reply:
(though either is valid)